.github/pull_request_template.md

```diff
@@ -37,16 +43,27 @@ How to test the changes in this Pull Request:
 1.
 2.
 3.
+
 ### User Facing Testing
```

Comment:
Do you know what's the difference between `Manual Testing` and `User Facing Testing`? I wonder if we can take the chance and unify those two sections.

Reply:
This is a great point. 
Manual testing was for the instructions from us, maintainers, this is where branch/code changes would go in order to test. etc
User Facing Testing would be the nicely, end-user facing testing steps would go. 

I guess we can unify and have only something along the lines of 
```
### Testing steps

<!-- Check if the change is not relevant to the "user" (merchant) or the testing requires branch / code changes -->
* [ ] Don't include in the release Testing Notes

<!-- If the above check is not made, then write these steps from the perspective of a "user" (merchant) familiar with WooCommerce. No need to spell out the steps for common setup scenarios (eg. "Create a product"), but do be specific about the thing being tested. Include screenshots demonstrating expectations where that will be helpful. -->

1.
2.
3.
```

Reply:
> I guess we can unify and have only something along the lines of

:100:

Reply:
@ralucaStan are you ok if I implement this change and merge `Manual Testing` and `User Facing Testing` into a `Testing steps` section?

Reply:
Sure, thanks for taking over here.

Reply:
Done in 45cc99075e62813bdefdee856922330f75a6ef36.

Reply:
Looks great 👍